index.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Apple</title>
+  <link rel="stylesheet" href="styles.css" />
+  <link href="https://www.fontify.me/wf/1cc89de3d33c62b9e9d8cdb0afa715aa" rel="stylesheet" type="text/css" />
+  <script src="https://kit.fontawesome.com/a076d05399.js"></script>
+</head>
+
+<body>
+
+  <div class="first-bg">
+    <nav>
+      <ul class="flex nav-links">
+        <li class="nav-items">
+          <a href=""><i class="fab fa-apple"></i></a>
+        </li>
+        <li class="nav-items"><a href="">Store</a></li>
+        <li class="nav-items"><a href="">Mac</a></li>
+        <li class="nav-items"><a href="">IPod</a></li>
+        <li class="nav-items"><a href=""> IPhone</a></li>
+        <li class="nav-items"><a href="">IPad</a></li>
+        <li class="nav-items"><a href="">ITunes</a></li>
+        <li class="nav-items"><a href="">Support</a></li>
+        <li class="nav-items"><input type="text" /></li>
```

Comment:
Great work with making these links. However, If you check the original webpage there is a hover effect once you hover the elements kindly add that